src/js/tracks/text-track.js

```diff
@@ -255,16 +218,14 @@ class TextTrack extends EventTarget {
       set() {}
     });
 
-    if (options.src) {
-      tt.src = options.src;
-      loadTrack(options.src, tt);
+    if (settings.src) {
+      tt.src = settings.src;
+      loadTrack(settings.src, tt);
     } else {
       tt.loaded_ = true;
     }
 
-    if (browser.IS_IE8) {
-      return tt;
-    }
+    return tt;
```

Comment:
I think we decided that it did not hurt anything to always return an instance of the object or on IE8 a dom object